kubernetes-workshop/6002_helm/modul_helm_6002_01_slides.md

```diff
@@ -321,7 +328,65 @@ metadata:
     app: mychart
 ```
 
-## Demo: Let's write some infra
+## Helm Hooks
+
+* Hook can be used to hook into the install, update and delete lifecycle
+* Hooks are regular templates
+* They are created using annotations
+* You should opt to put them into the `templates/hooks` subdir
+* Availability depends on helm version
+
+## Hook annotations
+
+```yaml
+metadata:
+  annotations:
+    "helm.sh/hook": "<hook-type>"
+```
+
+## Install Hooks
+
+* pre-install
+* post-install
+
+## Upgrade Hooks
+
+* pre-upgrade
+* post-upgrade
+
+## Rollback Hooks
+
+* pre-rollback
+* post-rollback
+
+## Delete Hooks
+
+* pre-delete
+* post-delete
+
+## CRD Hooks
+
+* crd-install
+
+Is used on Custom Resource Definitions to ensure that they are
```

Comment:
I had to read this three times to understand it right. Perhaps it's only me, but I would prefer a simpler sentence here.

Reply:
Change to the following which I'm assuming has a better readability:

> Used on Custom Resource Definitions to ensure that they are
> defined before they are used by other manifests in the chart.